github/resource_github_user_invitation_accepter.go

```diff
@@ -0,0 +1,57 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceGithubUserInvitationAccepter() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGithubUserInvitationAccepterCreate,
+		Read:   resourceGithubUserInvitationAccepterRead,
+		Delete: resourceGithubUserInvitationAccepterDelete,
```

Comment:
Nit: Should we use [`schema.Noop`](https://godoc.org/github.com/hashicorp/terraform/helper/schema#Noop) instead of making an empty, custom function?

Reply:
TIL `schema.Noop` is a thing.